### DISCUSSION TITLE
Package class not recognised Laravel 8

### DISCUSSION THREAD

**Discussion by Hishman** [spatie/laravel-backup#1323] [Q&A] [answered]
Hi,
I am trying to use the Laravel backup package but I keep getting the following error:
**_syntax error, unexpected 'Package' (T_STRING), expecting function (T_FUNCTION) or const (T_CONST)_**
It's about recognising the class Package in the PackageServiceProvider.php file. 

![image](https://user-images.githubusercontent.com/16415623/120458782-7a07c000-c38f-11eb-8e5e-cdf7d7795ef2.png)

I am using both Laravel 8 and PHP 8 in my Laravel app.

This is the composer.json file
![image](https://user-images.githubusercontent.com/16415623/120458987-acb1b880-c38f-11eb-9029-3a631fb7b8cf.png)

Any help is much appreciated.

**Comment by freekmurze** [comment, marked as answer]
It is part of the `laravel-package-tools` package, which should be pulled in by composer

https://github.com/spatie/laravel-backup/blob/39dc00988cd28e7cd06237dae27bf463fa9420fd/composer.json#L32

**Reply by Hishman** [reply]
Hi and thanks for quick reply. I am afraid I still get the same error even after installing laravel-package-tools.
![image](https://user-images.githubusercontent.com/16415623/120475616-81d06000-c3a1-11eb-8407-d1570c88308b.png)
![image](https://user-images.githubusercontent.com/16415623/120475783-b5ab8580-c3a1-11eb-8840-a4c9c3e5b71e.png)

**Reply by Hishman** [reply]
> It is part of the `laravel-package-tools` package, which should be pulled in by composer
> 
> https://github.com/spatie/laravel-backup/blob/39dc00988cd28e7cd06237dae27bf463fa9420fd/composer.json#L32

This answer doesn't help resolve the issue or lacks clear instructions. Please can anyone elaborate a bit more.

**Reply by pilishen** [reply]
i think you should double check your php version
what's the output of `php -v`

**Reply by pilishen** [reply]
> i think you should double check your php version what's the output of `php -v`

maybe due to some reason, your php 8 version wasn't loaded correctly, php 7.x could be still actually loaded
